types/applepayjs/tslint.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json",
```

Comment:
this changed recently; it needs to be `@definitelytyped/dtslint/dt.json`

CI was broken and should pass once it's re-run with this change reverted.